src/generator/generator.controller.js

```diff
@@ -89,6 +89,13 @@ exports.playbookPipeline = async function ({issues, auto_reboot = true}, remedia
         issues = addPostRunCheckIn(issues);
     }
 
+    // Add play that generates a new Compliance report when there are Compliance(ssg) issues  
+    const complianceIssues = _.filter(issues, issue => issue.id.app == 'ssg');
```

Comment:
Does it need to grab all compliance issues to see whether there is a one occurrence?

Reply:
Hmmm yeah we only need to check for one occurrence. I'll update this too. Thanks for the review!